caffe2/CMakeLists.txt

```diff
@@ -596,6 +596,10 @@ if(NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
     list(APPEND TORCH_SRCS ${LITE_EAGER_SYMOBLICATION_SRCS})
   endif()
 
+  list(APPEND TORCH_SRCS
+    ${TORCH_SRC_DIR}/csrc/utils/verbose_wrapper.cpp
+  )
+
```

Comment:
Why is it necessary to make a change to `CMakeLists.txt` rather than adding it to the appropriate list in `build_variables.bzl` (as it is already done with `verbose.cpp`)

This causes an internal build system linker error, as it is based on buck, rather than Cmake:
```
ImportError: ../libtorch_python.so: undefined symbol: _ZN5torch7verbose16_mkl_set_verboseEi
```